templates/dashboard/includes/_collection_availability.html

```diff
@@ -1,16 +1,17 @@
 {% load i18n %}
 
-  <div class="label label-{{ label_cls }}">
-    {% if is_visible %}
-      {% trans "Published" context "Collection availability status" %}
-    {% elif not collection.is_published %}
-    <div class="label">
-      {% trans "Hidden" context "Collection availability status" %}
-    </div>
-    {% elif collection.is_published and collection.published_date is not None %}
+<div class="label label-{{ label_cls }}">
+  {% if is_visible %}
+    {% trans "Published" context "Collection availability status" %}
+  {% elif not collection.is_published %}
```

Comment:
We could simplify the logic with:
```python
if is_visible:
    'published'
elif collection.published_date is not None: 
    '{date}'
else:
    'Hidden'
```

Reply:
What about case:
```python
collection.is_published = False and collection.published_date is not None
```
with simplified logic it will say that is hidden and will become available on published_date, but I think its misleading as is_published = False then it will not be published on published_date.
```python
    @property
    def is_visible(self):
        return self.is_published and (
            self.published_date is None or
            self.published_date <= datetime.date.today())
```

Reply:
> ```python
> collection.is_published = False and collection.published_date is not None
> ```

It's the same thing as the second case:

> ```python
> if is_visible: pass
> elif collection.published_date is not None: pass
> ```

I don't understand what you mean.

Reply:
I apologise for not making myself clear, that's correct that its same thing as the second case, but at least for me the message which is displayed in dashboard for this case is misleading when logic is simplified.
Lets take a look at the example, I simplified this logic as you said then created four collections in which different publication options were used. Below you can see what publication options were used for given collection:
1) Published without publication date
2) Published with publication date
3) **Not published with publication date**
4) Not published without publication date

Now in dashboard their visibility column shows:
1) Published 
2) Hidden, will become visible on date
3) **Hidden, will become visible on date**
4) Hidden

1st, 2nd and 4th is correct, but is 3rd correct? Should it say in visibility column that it will become visible when published field is false and published_date is set? In my opinion it shouldn't as is_visible model property still would be false even when published_date is reached. It can be fixed with nested if inside elif but that would same logic as is now.

I hope this time is more clear what I mean.